### DISCUSSION TITLE
New base shape - curve

### DISCUSSION THREAD

**Discussion by aheadweb** [pikasojs/pikaso#102] [Ideas]
Hello. In my project, I often use a curve. And it is the basis for complex figures. I have not the most successful [**example**](https://codesandbox.io/s/pikaso-curvedrawer-via-group-shape-ts-error-xtfrh9?file=/src/App.tsx) of its implementation. I propose to discuss how best to do it. Then I could open a pull request with a new shape added.

For example сurve manipulation handlers. Need to combine a curve and its handlers into a group?

**Comment by aheadweb** [comment]
@raminious

**Reply by raminious** [reply]
Hi @aheadweb 
I was away from this project for a while and I need to think about it to figure out what is the best way to implement that
